examples/dbclient/common/src/main/resources/META-INF/native-image/io.helidon.applications/helidon-examples-dbclient-common/reflect-config.json

```diff
@@ -0,0 +1,9 @@
+[{
+  "name" : "io.helidon.examples.dbclient.common.Pokemon",
```

Comment:
Just note for the future: Maybe we can implement some annotation (like JPA Enity) to mark those dbClient POJOs for mapping.

Reply:
Yes, I am thinking about. I think we should keep it post 2.0 though. I will create an issue for it

Reply:
Created #2030  for this